_posts/_2022-11-08-thudtest.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+authors: 
+  - hdhalter
+comments: true
+title: "The thud test: Measuring OpenSearch Documentation"
+category:
```

Comment:
insert line above this with the date we expect to publish - example:

date: 2022-11-10 01:01:01 -0700


Also "category" should be changed to "categories"